### PR TITLE
Pymcon branding

### PR DESCRIPTION
**Pull request by rowangayleschaefer**
Added PyMCon folder -> README.md contains branding guide for PyMCon. 
Added PyMCon color swatches. Colors and swatches pulled from website and logos. 

Multiple commits here are dealing with organization and formatting.

### CODE REVIEW

**Comment by canyon289** [comment]
cc @cuchoi for approval

File: pymcon/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# PyMCon Branding
```
Comment (canyon289) [inline pymcon/README.md:1]: Add a link to the assets in the pymcon_website_2022 folder to ensure folks can find everything

**Comment by canyon289** [comment]
These .DS_Store files should not be included in the PR

image

File: pymcon/README.md (hunk 1 above)
Comment (rowangayleschaefer) [reply pymcon/README.md:1]: Fixed both!

**Comment by rowangayleschaefer** [comment]
Fixed both!

**Comment by canyon289** [comment]
Thank you much!

**Pull request merged**